com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -776,6 +777,11 @@ internal void WriteNetworkVariableData(FastBufferWriter writer, ulong targetClie
                 if (canClientRead)
                 {
                     var writePos = writer.Position;
+                    // Note: This value can't be packed because we don't know how large it will be in advance
+                    // we reserve space for it, then write the data, then come back and fill in the space
+                    // to pack here, we'd have to write data to a temporary buffer and copy it in - which
+                    // isn't worth possibly saving one byte IFF the data is less than 63 bytes long...
```

Comment:
_Nit-pick or curious what it means:_
IFF? abbreviation or misspelled?

Reply:
Haha, sorry, using formal logical language... IFF is shorthand for "If And Only If"

https://en.wikipedia.org/wiki/If_and_only_if

I'll change it.

Reply:
done